api/server.spec.js

```diff
@@ -0,0 +1,52 @@
+const request = require('supertest');
+const server = require('./server');
+const db = require('../data/dbConfig')
+
+describe('testing the route handlers', () => {
+  describe('testing GET /games', () => {
+    test('should respond with status 200 if successful', async () => {
+      const response = await request(server).get('/games')
+
+      expect(response.status).toBe(200)
+    })
+    
+    test('should respond with a json object', async () => {
+      const response = await request(server).get('/games')
+
+      expect(response.type).toMatch(/json/i)
+    })
+    
+    test('should respond with an array of objects', async () => {
```

Comment:
Might want to double test this by adding some games and then `get`ting them to make sure, that they actually get returned